### PR TITLE
Remove '--io_type file' from performance regression workflow

### PR DESCRIPTION
**Pull request by popematt**
**Issue #, if available:**

None

**Description of changes:**

This should speed up the performance testing workflow by _at least_ one order of magnitude, and help contribute to the stability of the benchmark results, since the serde time is often (and especially with smaller data) dominated by the file system access time.

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: amazon/ionbenchmark/Command.py [hunk 1]
```diff
@@ -1,8 +0,0 @@
-from enum import Enum
-
-
-class Command(Enum):
```
Comment (tgregg) [inline amazon/ionbenchmark/Command.py:4]: Accidentally deleted?

File: amazon/ionbenchmark/Command.py [hunk 2]
```diff
@@ -1,8 +0,0 @@
-from enum import Enum
```
Comment (cheqianh) [inline amazon/ionbenchmark/Command.py:1]: QQ, do we still need COMMAND?

File: amazon/ionbenchmark/Command.py (hunk 1 above)
Comment (popematt) [reply amazon/ionbenchmark/Command.py:4]: Good catch.

It's such a simple PR, that I thought I could get away without checking myself before posting it. 😅
...but I was wrong.

**Review by cheqianh — COMMENTED** [review]
Seems that CI/CD failed for PYPY due to deletion of command.

File: amazon/ionbenchmark/Command.py (hunk 2 above)
Comment (popematt) [reply amazon/ionbenchmark/Command.py]: This was an accidental deletion. I have re-added it.

**Review by cheqianh — APPROVED** [review]

**Pull request merged**